pkg/client/images.go

```diff
@@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
 	err = c.r.Delete().Resource("images").Name(name).Do().Error()
 	return
 }
+
+// UpdateSignatures allows to modify signatures of given image.
+func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
```

Comment:
NamedCreater.  I don't know if someone removed the NamedUpdater equivalent
or not.

On Mon, Jun 13, 2016 at 1:32 PM, David Eads <notifications@github.com>
wrote:

> In pkg/client/images.go
> <https://github.com/openshift/origin/pull/9181#discussion_r66833393>:
>
> > @@ -61,3 +62,10 @@ func (c *images) Delete(name string) (err error) {
> >  	err = c.r.Delete().Resource("images").Name(name).Do().Error()
> >  	return
> >  }
> > +
> > +// UpdateSignatures allows to modify signatures of given image.
> > +func (c *images) UpdateSignatures(image *imageapi.Image) (result *imageapi.Image, err error) {
>
> Yeah I think the wrapper that makes sense and models what we'd do for end
> users. Ref to image, although we may want to make the signature name be
> the unique verification (I don't know that I want a special name for a
> signature).
>
> That makes sense. Does the RESTHandler include the requested name part of
> the URL anywhere in the context to avoid having to repeat myself in the API?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/9181/files/9ff4bf43548c758b6767b639b335681285fece48#r66833393>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABG_p1XYyE9Yqlv79b12At_iZl5lg_8Hks5qLZQ_gaJpZM4IvEFx>
> .
>